interview-questions.md

```diff
@@ -4,51 +4,62 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer:  Object destructuring is breaking up the stored data items to seperate and make accessible the values held within an object.
 
+  Researched answer: Object Destructuring is a little bit of syntactic sugar (https://en.wikipedia.org/wiki/Syntactic_sugar) that can be used to bind the properties of an object to a variable. Sometimes working with JavaScript objects can become repetitive in the sense that you need to write out the object name followed by the property name like so: objectName.propertyName. Using object destructuring, you can simply use propertyName to achieve the same result.
 
 2. What are the three arguments that .map() can accept?
 
-  Your answer:
+  Your answer: Arrays, objects, and variables
 
-  Researched answer:
+  Researched answer:  Rest parameters, default parameters or destructured parameters
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: A class definition can be passed / inherited. An object does not have this ability.
```

Comment:
a class is a blueprint for objects to be created from. objects are a way of housing data and behavior.